kie-dmn-jpmml/src/test/java/org/kie/dmn/jpmml/DMNInvokingjPMMLTest.java

```diff
@@ -115,6 +118,33 @@ public void testDummyInteger() {
         assertThat(DMNRuntimeUtil.formatMessages(dmnResult.getMessages()), dmnResult.hasErrors(), is(false));
 
         final DMNContext result = dmnResult.getContext();
-        assertThat((Map<String, Object>) result.get("hardcoded"), hasEntry("result", new BigDecimal(3)));
+        assertThat(result.get("hardcoded"), is(new BigDecimal(3)));
+    }
+
+    @Test
+    public void testMultipleOutputs() {
```

Comment:
BTW I think these changes in kie-dmn-jpmml module are already on master so should be excluded from this PR.